<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Touchscreen</class>
 <widget class="QMainWindow" name="Touchscreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Touchscreen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Status</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>9</y>
        <width>441</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Change plant</string>
     </attribute>
     <widget class="QComboBox" name="plant_combobox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="rotate_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="temperature_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="type_combobox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Change_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>95</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QSlider" name="moisture_slider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>106</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Plant pots</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>70</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Plant type</string>
      </property>
     </widget>
     <widget class="QPushButton" name="default_button">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>95</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Default</string>
      </property>
     </widget>
     <widget class="QLabel" name="moisture_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>120</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Moisture 50%</string>
      </property>
     </widget>
     <widget class="QLabel" name="temperature_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max temperature 100C</string>
      </property>
     </widget>
     <widget class="QLabel" name="rotate_label">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Rotate 0 deg/hour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>95</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>settings and test</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>201</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>status</string>
       </attribute>
       <widget class="QPushButton" name="get_status_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>95</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Get status</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="Status_msg">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>311</width>
          <height>151</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>&lt;-Click</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>test page</string>
       </attribute>
       <widget class="QPushButton" name="test_add">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>test ADD</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
